src/SFA.DAS.EmployerFinance.Web/Orchestrators/TransfersOrchestrator.cs

```diff
@@ -41,10 +41,11 @@ public async Task<OrchestratorResponse<IndexViewModel>> GetIndexViewModel(string
             var accountId = _hashingService.DecodeValue(hashedAccountId);
             var indexTask = _transfersService.GetCounts(accountId);
             var accountDetail = _accountApiClient.GetAccount(hashedAccountId);
+            var financialBreakdownTask = _transfersService.GetFinancialBreakdown(accountId);
 
             var renderCreateTransfersPledgeButtonTask = _authorizationService.IsAuthorizedAsync(EmployerUserRole.OwnerOrTransactor);            
 
-            await Task.WhenAll(indexTask, renderCreateTransfersPledgeButtonTask, accountDetail);
+            await Task.WhenAll(indexTask, renderCreateTransfersPledgeButtonTask, accountDetail, financialBreakdownTask);
```

Comment:
I may have misread this, but I thought that the indexTask was a call to an outer api endpoint, in which case the data we get from financialBreakdownTask ought to really come from there. There should be a 1:1 ratio between UI endpoints and outer api endpoints.